### PR TITLE
Fixed `PyTorchClassifier:predict_proba` to omit probability values

### PR DESCRIPTION
**Pull request by oroszgy**
There was a bug in the `predict_proba` method as calling the final `softmax` was missing

### CODE REVIEW

**Comment by aconneau** [comment]
Hi, thanks for using SentEval. In this particular case, we actually don't need a softmax here, because it's already in the "CrossEntropyLoss" ( http://pytorch.org/docs/master/nn.html#torch.nn.CrossEntropyLoss ) . If the loss had been the "NLLLoss" then indeed we would have needed a softmax before. 
Thanks

**Comment by oroszgy** [comment]
@aconneau Thanks for reviewing my PR. You are right, that none of the classification model needs an additional SoftMax layer. However, when you want to use your trained model, especially if you need probability values, then we need to run through the values over the softmax function. As far as I understood the code, `predict_proba` should omit probability values, that is why added the softmax calculation only to that method.

Can I ask you to take another look at the PR and if necessary could you please revise your decision?

Thanks in advance!

**Comment by aconneau** [comment]
You are right.

`vals = F.softmax(self.model(Xbatch).data.cpu().numpy())`

should be:

`vals = F.softmax(self.model(Xbatch).data.cpu()).numpy()`

instead? The output of the one above may be a torch Tensor, while below is a numpy array? (maybe it handles automatically, please confirm). Did you check with the simple bow.py script that it didn't introduce any bug?

**Comment by oroszgy** [comment]
@aconneau  sorry for the very late reply. The mentioned conversion is handled automatically and `examples/bow.py` runs flawlessly.
